paddle/operators/sequence_conv_op.cc

```diff
@@ -128,25 +128,25 @@ class SequenceConvOpMaker : public framework::OpProtoAndCheckerMaker {
         "this LoDTensor is a matrix with shape (T, D), where, T is the "
         "total time steps in this mini-batch, D is the output feature size.");
 
-    AddAttr<bool>("padding_trainable",
+    AddAttr<bool>("paddingTrainable",
                   "(bool, default false) the padding data of SequenceConvOp "
                   "is trainable or not.")
         .SetDefault(false);
-    AddAttr<int>("context_length",
-                 "(int, default 3) the context_length of SequenceConvOp is the "
+    AddAttr<int>("contextLength",
+                 "(int, default 3) the contextLength of SequenceConvOp is the "
                  "height of the convolution kernel.")
         .SetDefault(3)
```

Comment:
I think this is an argument the users must set, so please do not set the default value.